apps/docs/src/app/core/component-docs/link/examples/link-example.component.html

```diff
@@ -1,18 +1,21 @@
-<a [routerLink]="['./']" fd-link>Standard Link</a> <br /><br />
-<a [routerLink]="['./']" fd-link [emphasized]="true">Emphasized Link</a>
+<a [routerLink]="['./']" fd-link aria-label="Standard link">Standard Link</a> <br /><br />
+<a [routerLink]="['./']" fd-link [emphasized]="true" aria-label="my Link">Emphasized Link</a>
```

Comment:
Shouldn't the `aria-label` be "Emphasized Link"?

Reply:
fixed